From e6b9baa2b600e7bb73a0c2a285e35005d347161e Mon Sep 17 00:00:00 2001
From: Jose Luis Blanco-Claraco <joseluisblancoc@gmail.com>
Date: Mon, 5 Oct 2020 07:38:17 +0200
Subject: [PATCH] fix debian/rules multiarch paths

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 111d548..94b9114 100755
--- a/debian/rules
+++ b/debian/rules
@@ -51,7 +51,7 @@ override_dh_shlibdeps:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/melodic/setup.sh" ]; then . "/opt/ros/melodic/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/ros-melodic-mrpt2//opt/ros/melodic/lib/ -l$(CURDIR)/debian/ros-melodic-mrpt2//opt/ros/melodic/lib/${DEB_HOST_MULTIARCH}
+	dh_shlibdeps -l$(CURDIR)/debian/ros-melodic-mrpt2//opt/ros/melodic/lib/:$(CURDIR)/debian/ros-melodic-mrpt2//opt/ros/melodic/lib/${DEB_HOST_MULTIARCH}
 
 
 override_dh_auto_install:
-- 
2.17.1

